BlueprintUICommonControls/Sources/TransitionContainer.swift

```diff
@@ -59,7 +59,7 @@ public struct TransitionContainer: Element {
     }
 
     public func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription? {
-        return UIView.describe { config in
+        return PassthroughView.describe { config in
```

Comment:
Thinking out loud: Any other view-backed elements that use UIView we should use this new type for? cc @watt

Reply:
> Thinking out loud: Any other view-backed elements that use UIView we should use this new type for? cc @watt

Might be a good base class for the accessibility related elements that are fully transparent containers? Nothing else comes to mind.